controllers/admin/AdminBlockListingController.php

```diff
@@ -45,46 +44,53 @@ public function ajaxProcessChangeBlockStatus()
 
         $updateResult = Db::getInstance()->update('psreassurance', $dataToUpdate, $whereCondition);
 
-        $this->ajaxDie(json_encode($updateResult ? 'success' : 'error'));
+        // Response
+        header('Content-Type: application/json');
```

Comment:
create a dedicated function instead.
```php
protected function renderJson($content)
{
  header('Content-Type: application/json');
  $this->ajaxRender($content);
}
```

Reply:
May be an another solution : In ControllerCore, add a method : 

```php
protected function ajaxRenderJson($content)
{
  header('Content-Type: application/json');
  $this->ajaxRender(json_encode($content));
}
```

What do you think about it ?

Reply:
Yes, but you'll not be able to release the module until the Core release a 1.7.7.0 :/

Reply:
Correct ! I create this method in the controller also.